src/lib/transform.js

```diff
@@ -246,7 +246,7 @@ function makeGoalsAndObjectivesObject(objectiveRecords) {
       goal, title, status, ttaProvided, topics, files, resources,
     } = objective;
     const goalId = goal ? goal.id : null;
-    const titleMd5 = md5(title);
+    const titleMd5 = md5(title || '');
```

Comment:
if we are hitting this, then there is a bad join case somewhere. and the objective is getting excluded somehow

Reply:
yeah Nathan already fixed this, I can remove the ternary